channel.rb

```diff
@@ -14,7 +14,7 @@ def initialize(channel, domain)
     @domain = domain
     @timestamp = Time.now
     @lock = Concurrent::ReadWriteLock.new
-    reload
```

Comment:
I think this was basically refetching each channel on boot, which blocked requests for a few minutes.